<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersListDialog</class>
 <widget class="QDialog" name="UsersListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Users</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>383</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="choose">
     <property name="text">
      <string>Choose</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListWidget" name="usersList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
